kube/app.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: wattx-gke-snapshooter
+spec:
+  replicas: 1
```

Comment:
according to the https://kubernetes.io/docs/concepts/workloads/controllers/deployment/ it's 2 by default